notebooks/how_to/run_unit_metrics.ipynb

```diff
@@ -0,0 +1,1369 @@
+       "7968                0.0              0.0                 0.176599          0.0  \n",
+       "102                 0.0              1.0                 0.262595          0.0  \n",
+       "7674                0.0              0.0                 0.114304          0.0  \n",
+       "3920                1.0              0.0                 0.228684          0.0  \n",
+       "...                 ...              ...                      ...          ...  \n",
+       "6269                0.0              0.0                 0.146433          0.0  \n",
+       "5872                0.0              0.0                 0.264931          0.0  \n",
+       "2967                1.0              0.0                 0.625799          1.0  \n",
+       "1093                0.0              0.0                 0.142927          0.0  \n",
+       "1633                0.0              1.0                 0.131974          0.0  \n",
+       "\n",
+       "[1600 rows x 15 columns]"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vm_test_ds._df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running Unit Metrics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Computing F1 Score \n",
+    "\n",
+    "The following snippet shows how to set up and execute a unit metric implementation of the F1 score from `sklearn`. In this example, our objective is to compute F1 for the test dataset. Therefore, we specify `vm_test_ds` as the dataset in the inputs along with the `metric_id`. \n",
+    "\n",
+    "**Dataset to Metric Input Mapping**\n",
+    "\n",
+    "To accurately compute the F1 score, it's essential to ensure that these columns are correctly aligned and contain the relevant data. The F1 score requires two inputs: \n",
+    "\n",
+    "- the predictions `y_pred` and \n",
+    "- the true labels `y_true`\n",
+    " \n",
+    "By selecting `vm_test_ds`, we include the columns necessary for calculating the F1 score from `sklearn`. These are `y_pred`, the predictions mapped to the `vm_test_ds.prediction_column`, in this case labeled as 'Predictions', and the true label column `vm_test_ds.target_column` named 'Exited'. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Computing metric value for 'validmind.unit_metrics.sklearn.classification.F1'\n",
+      "y_pred obtained from pre-computed predictions in dataset column 'Predictions' from 'my_model'\n",
+      "y_true obtained from column 'Exited'\n"
+     ]
+    }
+   ],
+   "source": [
+    "metric_id = \"validmind.unit_metrics.sklearn.classification.F1\"\n",
+    "\n",
+    "inputs = {\"dataset\": vm_test_ds}\n",
```

Comment:
As discussed today, `model` is a required input as well. We can use `y_pred(model_id=...)` to access the predictions for a given model.

Reply:
Addressed.